cmr/queries.py

```diff
@@ -562,11 +622,11 @@ def polygon(self, coordinates: Sequence[PointLike]) -> Self:
         return self
 
     def bounding_box(
-        self,
-        lower_left_lon: FloatLike,
-        lower_left_lat: FloatLike,
-        upper_right_lon: FloatLike,
-        upper_right_lat: FloatLike,
+            self,
+            lower_left_lon: FloatLike,
+            lower_left_lat: FloatLike,
+            upper_right_lon: FloatLike,
+            upper_right_lat: FloatLike,
```

Comment:
I think extra indentation slipped in via another PR:

```suggestion
        self,
        lower_left_lon: FloatLike,
        lower_left_lat: FloatLike,
        upper_right_lon: FloatLike,
        upper_right_lat: FloatLike,
```